vendor/woocommerce/action-scheduler/classes/ActionScheduler_DataController.php

```
@@ -0,0 +1,210 @@
+		if ( self::$free_ticks && 0 === $ticks % self::$free_ticks ) {
+			self::free_memory();
+		}
+	}
+
+	/**
+	 * Reduce memory footprint by clearing the database query and object caches.
+	 */
+	public static function free_memory() {
+		if ( 0 < self::$sleep_time ) {
+			/* translators: %d: amount of time */
+			\WP_CLI::warning( sprintf( _n( 'Stopped the insanity for %d second', 'Stopped the insanity for %d seconds', self::$sleep_time, 'action-scheduler' ), self::$sleep_time ) );
```

PHPStan (2.1.46)

[error] Call to static method warning() on an unknown class WP_CLI.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)